pinot-core/src/main/java/com/linkedin/pinot/core/segment/index/ColumnMetadata.java

```diff
@@ -262,7 +281,7 @@ public ColumnMetadata build() {
   private ColumnMetadata(String columnName, int cardinality, int totalRawDocs, int totalAggDocs, int totalDocs, DataType dataType, int bitsPerElement,
       int stringColumnMaxLength, FieldType fieldType, boolean isSorted, boolean hasInvertedIndex,
       boolean insSingleValue, int maxNumberOfMultiValues, boolean hasNulls, boolean hasDictionary, TimeUnit timeunit,
-      int totalNumberOfEntries) {
+      int totalNumberOfEntries, char paddingCharacter) {
```

Comment:
The padding character is used to pad Strings which are strings of char's. So, the padding character is also a char. char is a 16-bit character.